routes/api/users.js

```diff
@@ -0,0 +1,81 @@
+const router = require("express").Router();
+const passport = require("passport");
+
+router.get("/user", (req, res, next) => {
+  User.findById(req.payload.id)
```

Comment:
Please delete this routes folder outside the server, we do not need it anylonger

Reply:
Yes!